Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
+MAINTAINER Yi Wang <yi.wang.2005@gmail.com>
+
+COPY build.sh /opt
```

Comment:
You probably already know, just FYI: During docker build process, it copies all files in directory running the docker build command to docker context. Which will cost long time if directory is too big. In that case, you can use `.dockerignore` file to ignore copying any file. https://docs.docker.com/engine/reference/builder/